app/views/search/_attachment.html.erb

```diff
@@ -1,11 +1,15 @@
-<div class="title">
-  Attachment on story <%= link_to model.story.name, [model.story.project, model.story] %> in <%= link_to model.story.project.name, model.story.project %>
-</div>
+<% if model.story.present? %>
```

Comment:
Yeah I think we should add our own method, it shouldn’t really be a concern of the view layer to check if data should be displayed or not, it should simply never make it to the view layer.

We should probably add some unit tests to ensure when searching that comments from archived stories don’t show up